modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -29,12 +29,10 @@ Callback authCallback = new Callback<UserPrincipal>() {
 };
 ----
 
-. Retrieve an instance of the `AuthService` and use the `login` method to hand over to the login browser:
+. Use the `login` method to hand over to the login browser:
 +
 [source,java]
 ----
-AuthService authService = MobileCore.getInstance().getService(AuthService.class);
```

Comment:
After removing this code is not going to `compile` I know it's snippet but best to still create this service from singleton.

Reply:
There is no singleton because `AuthService` creation requires an `AuthServiceConfiguration` provides by the developer.